src/seedu/addressbook/data/AddressBook.java

```diff
@@ -38,6 +38,12 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
         allPersons.add(toAdd);
     }
 
+    /**
+     * Sort the list by name
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)